### ISSUE TITLE
About train_dataset and val_dataset.

### ISSUE THREAD

**Issue by BigConsin** [NiFangBaAGe/CoordFill#2]
Hi,thanks for your great job here,and share your amazing codes here.
I am little confused about your dataset in your paper,said "We use the irregular mask from the previous method (Liu et al. 2018) for mask generation with holes up to 25% following HiFill (Yi et al. 2020)." But i can not find the setting in these  papers or their codes.Would you please share your train and test dataset details? That would really helps us a lot.
Looking forward to your reply!

**Comment by NiFangBaAGe** [comment]
Thanks for your interesting in our work.
The irregular mask (Liu et al. 2018) can be found at https://nv-adlr.github.io/publication/partialconv-inpainting. And the setting of mask ratio for high-resolution image inpainting can be found in Section 4.3 (Yi et al. 2020). As for the code, you can generate custom dataset via creating a mask-generator function using OpenCV to generate the desired mask.

**Comment by BigConsin** [comment]
total understand,thanks lot!
